Aliases/br

```diff
@@ -0,0 +1 @@
+../Formula/brooklyn-client.rb
```

Comment:
> That's even longer. We don't need (or usually add) an alias for a new formula so let's just skip one for this PR, thanks.

Ah sorry should have been clearer, I'm nuking the alias and renaming the class/file